microprofile.jdt/org.eclipse.lsp4mp.jdt.core/src/main/java/org/eclipse/lsp4mp/jdt/core/java/definition/AbstractAnnotationDefinitionParticipant.java

```diff
@@ -135,7 +135,6 @@ public boolean isAdaptedForDefinition(JavaDefinitionContext context, IProgressMo
 		// Get the annotation member value range
 		ISourceRange r = ((ISourceReference) annotation).getSourceRange();

```

Comment:
Do we need this code now, since you have annotationMemberInfo?

Reply:
The offsets are different (e.g. this range encompasses the ""  where as the annotationMemberInfo one doesn't). Should we change the definition range to leave out the "", so then we can simply use getRange like hover does?

Reply:
I think we should centralize the range compute for hover and definition in the same place. I think it is better to leave definition range without quote. I wonder if hover could follow this behavior too? If it is not possible or If the result is not vers good i suggest you that you add à withQuote boolezn parameter in getmembervalueat

Reply:
The hover range leaves out the quotes, so to be consistent with hover, we can do that with definition as well.

Reply:
Otherwise we can just set withQuote to true in definition for now

Reply:
I would prefer having à range definition without the quote
 
To have thé similar behavior with html href hyperlink in vscode.

What is rhe rzsult of rhe hover if range has no quote?

Reply:
Seems to behave the same as with quote, the only difference is the range on the tests. The quotes do not show on the vscode side regardless.